src/index.html

```diff
@@ -7,7 +7,17 @@
     <title>Development</title>
   </head>
   <body>
-    <header>Leaderboard</header>
+    <section class="top">
+      <img
+        src="https://media.tenor.com/3klZkDif0nsAAAAd/gaming-gif.gif"
+        alt="gaming"
+      />
+      <header>Leaderboard</header>
+      <img
+        src="https://media.tenor.com/gTcs1tOaswIAAAAd/asix-brotherhood-gaming.gif"
+        alt="young"
+      />
+    </section>
     <section class="container">
```

Comment:
- [ ] Your submit button is not properly aligned as per the given [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). Kindly not that we want the design to match the layout of the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), therefore kindly ensure that the submit button is aligned to the right of the form.
**Your design:**
![image](https://user-images.githubusercontent.com/12745474/202714523-01acc768-5a83-4a2b-876d-fafb75b09762.png)

**Wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/202715257-f6f19d43-7229-4e1c-a251-c6d91357fa54.png)